l10n_de_tax_statement/models/l10n_de_tax_statement_2019.py

```diff
@@ -138,99 +132,87 @@
-    },
-    '55': {
-        'code': '55', 'tax': 0.0,
         'name': _('Vorsteuerbeträge aus Rechn. v.a. Unternehmen g. § 15 '
                   'Abs. S. 1 Nr. 1 UStG a. Leistungen i.S.d. § 13a '
                   'Abs. 1 Nr. 6 UStG u. § 15 Abs. 1 S. 1 Nr. 5 UStG '
                   'u. a. innerg. Dreiecksgesch. g. § 25b A. 5 UStG (66)'),
     },
-    '56': {
-        'code': '56', 'tax': 0.0,
+    '54': {
+        'code': '54', 'tax': 0.0,
         'name': _('Vorsteuerbeträge a. d. innerg. Erwerb v. Gegenständen '
                   'gem. § 15 Abs. 1 Satz 1 Nr. 3 UStG (61)'),
     },
-    '57': {
-        'code': '57', 'tax': 0.0,
+    '55': {
+        'code': '55', 'tax': 0.0,
         'name': _('Entst. Einfuhrumsatzst. g. § 15 Abs. 1 S. 1 Nr. 2 '
                   'UStG (62)'),
     },
-    '58': {
-        'code': '58', 'tax': 0.0,
+    '56': {
+        'code': '56', 'tax': 0.0,
         'name': _('Vorsteuerbeträge aus Leistungen i. S. des § 13b UStG'
                   'i.V.m § 15 Abs. 1 Satz 1 Nr. 4 UStG (67)'),
     },
-    '59': {
-        'code': '59', 'tax': 0.0,
+    '57': {
+        'code': '57', 'tax': 0.0,
         'name': _('Vorsteuerbeträge d. n. allg. Durchschnittssätzen '
                   'berechnet sind gem. §§ 23 und 23a UStG (63)'),
     },
-    '60': {
-        'code': '60', 'tax': 0.0,
+    '58': {
+        'code': '58', 'tax': 0.0,
         'name': _('Berichtigung des Vorsteuerabzugs g. § 15 a UStG (64)'),
     },
-    '61': {
-        'code': '61', 'tax': 0.0,
+    '59': {
+        'code': '59', 'tax': 0.0,
         'name': _('Vorsteuerabzug f. innergem. Lief. neuer Fahrzeuge '
                   'außerh. e. Untern. g. §2a UStG sow. v. Kleinunt. i.S. '
                   'd. § 19 Abs. 1 i.V.m. § 15a Abs. 4a UStG (59)'),
     },
-    '62': {
-        'code': '62', 'tax': 0.0,
+    '60': {
+        'code': '60', 'tax': 0.0,
         'name': _('Verbleibender Betrag'),
     },
-    '63': {
-        'code': '63',
+    '61': {
+        'code': '61',
         'name': _('Andere Steuerbeträge'),
     },
-    '64': {
-        'code': '64', 'tax': 0.0,
+    '62': {
+        'code': '62', 'tax': 0.0,
         'name': _('Steuer inf. Wechsels d. Besteuerungsf. sow. Nachst. '
                   'a. verst. Anzahlungen u.a. wg. Steuersatzänd. (65)'),
     },
-    '65': {
-        'code': '65', 'tax': 0.0,
+    '63': {
+        'code': '63', 'tax': 0.0,
         'name': _('In Rechnungen unrichtig oder unberechtigt ausgewiesene '
                   'Steuerbeträge gem. § 14c UstG) sowie Steuerbetr. d. n. '
                   '§ 6a Abs. 4 S. 2, § 17 Abs. 1 S. 6, § 25 b Abs. 2 UStG '
                   'o. v. e. Auslagerer o. Lagerh. n. § 13a Abs. 1 Nr. 6 '
                   'UStG geschuldet werden (69)'),
     },
-    '66': {
-        'code': '66', 'tax': 0.0,
+    '64': {
+        'code': '64', 'tax': 0.0,
         'name': _('Umsatzsteuer-Vorauszahlung'),
     },
-    '67': {
-        'code': '67', 'tax': 0.0,
+    '65': {
```

Comment:
Is line 65 to be mapped with tag 39? If this is the case, the mapping is also missing in old version 2018.

Reply:
This line is special and should be used only in December and the value should be entered manually. Therefore i assume in 2018 version it was line 67.